lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb

```diff
@@ -56,14 +59,24 @@ def fetch_config
         })
 
       begin
-        Date.parse(@config_last_modified)
+        # Blind parse the LM string to check if it's a valid date.
+        # This short circuits the rest of the checks if it's not set
+        stored_date = Date.parse(@config_last_modified)
         if @config_last_modified != ""
-          req.options[:headers]["If-Modified-Since"] = Time.httpdate(@config_last_modified)
+          if min_last_modified != -1
+            parsed_sse_ts = Time.at(min_last_modified)
+            if parsed_sse_ts.utc > stored_date.utc
```

Comment:
Yes - that's what the slow polling time saves - we aren't retrying for that reason

Reply:
but shouldn't it retry until it gets the newer config if its told there is one, rather than waiting the full 10 minutes or whatever? that kind of makes realtime updates not useful if the SDK isn't going to necessarily get the update in a timely manner

Reply:
We can force a retry - but do we then not fall into the infinite retry problem? We can do the retry - but the likelihood on getting a stale config after SSE is pretty low - none of our testing in TL had it happen